index.html

```diff
@@ -149,9 +149,16 @@ <h1>Basic usage</h1>
 	<pre><code>&lt;pre>&lt;code class="language-css">p { color: red }&lt;/code>&lt;/pre></code></pre>
 	<p>If you use that pattern, the <code>&lt;pre></code> will automatically get the <code>language-xxxx</code> class (if it doesn’t already have it) and will be styled as a code block.</p>
 
-	<p>If you want to prevent any elements from being automatically highlighted, you can use the attribute <code>data-manual</code> on the <code>&lt;script></code> element you used for prism and use the <a href="extending.html#api">API</a>.
+	<p>If you want to prevent any elements from being automatically highlighted and instead use the <a href="extending.html#api">API</a>, you can set <code>Prism.manual = true;</code> before <code>DOMContentLoaded</code> is fired. By setting the attribute <code>data-manual</code> on the <code>&lt;script></code> element you used for prism this will be done automatically.
 	Example:</p>
 	<pre><code>&lt;script src="prism.js" data-manual>&lt;/script></code></pre>
+	<p>or</p>
+	<pre><code>&lt;script>
+window.Prism = window.Prism || {};
+window.Prism.manual = true;
```

Comment:
I know that this is what was suggested in the issue thread you linked to but we know that `highlightAutomaticallyCallback` is always called synchronously, so why don't we use this fact.

```html
<script src="prism.js"></script>
<script>Prism.manual = true;</script>
```

Shouldn't this work as well?

Reply:
Yes, I believe it should for a simple script tag. However I prefer the current version, because your suggestion will break if e.g. `async` is put onto the script tag or Prism is loaded by a loader such as require.js or head.js.

Reply:
That's true.